### PR TITLE
Update fair_roulette.md

### PR DESCRIPTION
**Pull request by JannemanDev**
syntax added

# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the [contribution guidelines](https://wiki.iota.org/smart-contracts/contribute) for this project
- [ ] I have performed a self-review of my own code
- [ ] I have selected the `develop` branch as the target branch
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/guide/example_projects/fair_roulette.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ All state changes such as the `round started` ,`round ended`, `placed bets`, and
 
 #### Building the Contract
 
-```
+```bash
```
Comment (howjmay) [inline documentation/docs/guide/example_projects/fair_roulette.md:134]: Should we use `shell` uniformly here?

Comment (lmoe) [reply documentation/docs/guide/example_projects/fair_roulette.md:134]: It's definitely not aligned right now. We have more `bash` than `shell` in the docs. I would say we merge the PR, decide on something and replace everything to either of them.

Afaik bash/zsh is just an alias for shell in markdown, so I would choose shell as bash/zsh is just that.

Comment (JannemanDev) [reply documentation/docs/guide/example_projects/fair_roulette.md:134]: You could use my markdown codeblock scanner to easily identify all codeblock and used types. It also gives some overall stats, see https://github.com/JanOonk/ScanMarkdownFilesForCodeSnippets

Comment (lmoe) [reply documentation/docs/guide/example_projects/fair_roulette.md:134]: Nice tool. Refreshing to see some .NET again. I will take a closer look into it today or tomorrow. Thanks :)

**Review by lmoe — APPROVED** [review]

**Pull request merged**